commands/pack.js

```diff
@@ -31,6 +31,7 @@ function displayHelp() {
 	console.log();
 	console.log('  Options');
 	console.log('    -o, --output      Specify an output directory');
+	console.log('    --content-hash    Add a unique hash to output files based on the content of an asset');
```

Comment:
Let's make this clearer.
```suggestion
	console.log('    --content-hash    Add a unique hash to output file names based on the content of an asset');
```